main/integration-tests/support/setup.js

```diff
@@ -82,13 +83,13 @@ class Setup {
 
   async getDefaultIndexId() {
     const adminSession = await this.defaultAdminSession();
-    const defaultProject = await adminSession.resources.projects.project(this.gen.defaultProjectId()).get();
+    const defaultProject = await adminSession.resources.projects.project(this.defaults.projectId).get();
     return defaultProject.indexId;
   }
 
   async getDefaultAwsAccountId() {
     const adminSession = await this.defaultAdminSession();
-    const defaultIndex = await adminSession.resources.indexes.index(this.defaultIndexId).get();
+    const defaultIndex = await adminSession.resources.indexes.index(this.defaults.indexId).get();
```

Comment:
Doing this.defaults.indexId (inside this function) does require that getDefaultAwsAccountId() be called at a specific sequence when building the defaults. I would suggest that you make the function as follows:
`async getDefaultAwsAccountId(indexId) { ... }`
This will eliminate the need to use this.defaults when we are still initializing it.

Reply:
Combined in one method as discussed